src/Features/Core/Portable/Completion/CompletionServiceWithProviders.cs

```diff
@@ -557,10 +563,10 @@ public TestAccessor(CompletionServiceWithProviders completionServiceWithProvider
                 => _completionServiceWithProviders = completionServiceWithProviders;
 
             internal ImmutableArray<CompletionProvider> GetAllProviders(ImmutableHashSet<string> roles)
-                => _completionServiceWithProviders._providerSource.GetProviders(roles, waitUntilAvaialble: true);
+                => _completionServiceWithProviders._providerSource.GetProvidersAsync(roles).Result;
```

Comment:
.Result can throw aggregate exceptions.  do .GetAwaiter().GetResult().  (but don't do this period :)).